frontend/src/pages/questionGroup/BarGraph.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react';
+import Paper from '@material-ui/core/Paper';
+import { makeStyles } from '@material-ui/core/styles';
+import { Bar } from 'react-chartjs-2';
+import Title from '../_shared/Title';
+
+const useStyles = makeStyles((theme) => ({
+  histogram: {
+    display: 'flex',
+    justifyContent: 'center',
+    component: 'h2',
+    variant: 'h6',
+    textAlign: 'center',
+  },
+}));
+
+const calculateDistribution = (
+  questionGroupAttempts,
+  amountQuestions,
+  questionIds
+) => {
+  let correctPerQuestion = Array.apply(null, Array(amountQuestions)).map(
```

Comment:
heb zelf nog nooit een Array.apply gebruikt, wat gebeurd hier precies?

Reply:
array vullen met 0'en ter groottte van amountQuestions (was de eerste hit op Google , maar kon niet zo snel vinden hoe je simpel [0 in range(amountQuestions)] moest doen in js

Reply:
new Array(amountQuestions).fill(0); it is

Reply:
@Josien94 je kan hier volgens mij zelfs de `new` keyword weghalen